app/main.py

```diff
@@ -1,6 +1,27 @@
+import os
+
+import requests
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    CITY = "Paris"
+    API_KEY = os.environ["API_KEY"]
+    URL = "https://api.weatherapi.com/v1/current.json"
```

Comment:
move these constants out of function. Wouldn't it be better if constant values are in global scope?

Reply:
Done )